lib/project_types/script/script_project.rb

```diff
@@ -27,6 +27,11 @@ def create(dir)
         FileUtils.mkdir_p(dir)
         Dir.chdir(dir)
       end
+
+      def cleanup(ctx:, script_name:, cli_dir:)
```

Comment:
nitpick: I don't really love the `cli_dir` param name. I'm a bad namer myself, but what about just naming it `root_dir`? Or something else?

And we should probably add a test for this functionality too